pkg/reconciler/route/traffic/rollout.go

```diff
@@ -270,14 +271,21 @@ func (cur *Rollout) Step(ctx context.Context, prev *Rollout, nowTS int64) (*Roll
 // If new percentage is larger than the previous, the last revision gets
 // the difference, if it is decreasing then we start removing traffic from
 // the older revisions.
-func adjustPercentage(goal int, cr *ConfigurationRollout) {
+func adjustPercentage(goal int, cr *ConfigurationRollout, logger *zap.SugaredLogger) {
+	// Usually changing traffic target %, especially during rollout is a rare
+	// occurrence, so log all 3 meaningful cases.
 	switch diff := goal - cr.Percent; {
 	case goal == 0:
+		logger.Infof("Traffic for config %s got assigned 0%%", cr.ConfigurationName)
```

Comment:
Use %q instead of %s?

Reply:
No, I hate those :) Do we need to disambiguate?